From 8ef77bd259131aa2b5d3bcefeac92e74c0167ee4 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Fri, 23 Oct 2026 10:26:41 +0200
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh 0x8E8B898CBF2412F9 from keyservers/WKD: same certificate(s), with self-signatures our copy was missing.

Assisted-by: Claude Opus 5.5
---
 0x8E8B898CBF2412F9  | 282 +++++++++++++++++++++++++++-----------------
 fcitx5-libthai.spec |   4 +-
 2 files changed, 174 insertions(+), 112 deletions(-)

diff --git a/0x8E8B898CBF2412F9 b/0x8E8B898CBF2412F9
index d9449d8..43b0e60 100644
--- a/0x8E8B898CBF2412F9
+++ b/0x8E8B898CBF2412F9
@@ -1,112 +1,174 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
-Version: SKS 1.1.6
-Comment: Hostname: carles.tubio.sks-database_1
 
-mQENBEyUvwkBCAD0P0fGutq6L7iixhhOMCaxvTwmT84oEcNP816hNo2+YSEws8Xp6uv94czW
-DyC1ibzh++mStReE7RioieiKtH/sYrfS1M2Gwv3kGYzekx5qKx92ncztEd08ufho3s3Qcw4Q
-V0IT8JLdnqCWOTHwcPeyLJ0ZZH9fseqHfrlsbVIlFI7BnMZyTW4xTvgX1jJ9A5m5JWW70iaO
-X+Yom1k5WRUDm8RtzzK0IAkdXwYHbrVCRNumbQK5ZtNZr3EmBxDt3lzpzD1Ow1EStRpVgd0Y
-VzHtv5N5o46Tp0L9j1br64zUbiOPrRsspVhpo0w9Mx70ay8JCBZqSBGM8GOSpsyP6g0FABEB
-AAG0H1dlbmcgWHVldGlhbiA8d2VuZ3h0QGdtYWlsLmNvbT6JARwEEAEIAAYFAlY51jwACgkQ
-mI3uMhX62HQvewf+PYNTp9CEEC38kB4FmHhMdwF4FCqnkAJN9V1INw2q07Z1fpZSiDddTcPf
-6MvOtZ3TmTAjtBes5q5UilJwfN8l+7BJA2KBV5vVVuIAEhu0qzcoxPJNhu7XJD7xZH1xoGc4
-0pnUwfc2pcTj92+HSrdI8NAyUC0mtBjRs8IV5yLXVfmGzxZECv7R6f50Wh2eM8xPEzrkIPJb
-pSb/OmYvmUAUJ5o0xU9YPPmAA4jmHeIkSoXJvV30uOULt/a0Ygk0Qx6COpnLRFrc6JZ+0LY1
-Sye82tqC6RQfegvvlCRNZEOGDu6eCX9ZYeHaXA6kin6xXDXKp+2jPL7Wf6I7iqVLKBJfdokB
-OAQTAQIAIgUCTJS/CQIbAwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AACgkQjouJjL8kEvna
-4AgAwPO68XQdpy4I8vF2t0/QcPUofxCw7Me4owBfnKDV4QaZ6oIIb+SJVBV3f5HmuaVeClXq
-+X0WgIOVa5gliVswsEgLANfP+nUYmq4W1oCqthoBKD/Qv7dlTLSbW+0o4NjEG5wqI/JJboaw
-SmvE1ZctZYwxyJb1SYYCQB8RBtMZqrGwXg3jr3HZwQnqJpSutFPTdCeNqajanVzLaDzebqWI
-+3UKDcaliPULhWFCkRx0tJi5m/LPe1X64VmLWcgj9aQ6FDlrnNrn8jGo/hegXAfWmb4K9zgp
-Fpw1tAUC9CRVs6UP4C5vY37ZrNpgFjGS9bE2Pg2spglaiolliQXjJlcV9YkBOwQTAQIAJQIb
-AwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AFAk9k/DkCGQEACgkQjouJjL8kEvkQMgf/YfSd
-mpnyxjA7VagXqR+/EkEOi2y5jEM4BepDwzNLXpcUdq8jIWN5icVAcNtMcbMws8BNKYRvweMI
-WAR+8cgliNfnu59kjua30qDv2ONEUijrknC+0VhXFQfO0dAKNgIs5TwSqeboUqSCmNDXEvlA
-uc98Ft61Ri/kuTYLpcm2E4cQ7+gAK+3T3alTZCC7l4y2QwGKFoLPC+n16YgePLRok74/DUHb
-Haml92ZFg8/LF3YWOIXUkR70TlQgSwoykXiouJKo1C6UQ6r0OH/5ersen4PL3PwvExlgzA4A
-ibAQv0E7Q+8nn+mJSEryK3Jp8FrRFKiMgcdip/iC3XZvZpDZsokCHAQQAQIABgUCUZeDcAAK
-CRADFyRkpJbN4HNJEACH+Vbdn5iJcwI41xc2x6ZiRR/FXXymNYuGtDd2NJlFuaL22fyxyacE
-WdD7t70l6diGCWVvR/696TjeJQojPOkTERAT++34RxdlM5ZTSah0JUWfzgvoTL9KoD0InYA+
-ALraX3zZHOKTf5jnhyqwujGXi3wunntF4jcxw5fMk+U3mfpMwniVs9o0BOVmfRWMjia4TTPr
-WUbizVY/hhbOXgUsHdYVj0GbLAvSj8IUVY/XerM+Zx/bZ54Hto9A5h8eo6jo5jIJOrAClh+V
-gh4Ind7V9jgq1Upxi3fKp2t/D2ZZu/hib5qJKorAW3Sw2gZxK4gNv1AdBNPKfgYk7HHZP/kJ
-KNwtl3pv/EE/R0oFxVaUHCOxe4GzIHHNXWxDCKUS5kElLhGnxFG//ZmpZ2Kya6+4FcsZVxNS
-tEoPR/DUtdtRMDO36DTMoDVHqIXaMJT250H24sKwB0zKD2vVKcvAisQdwC+hmLCAFV8c30aA
-2ijKlQ3HTqR08r54eRopO5T9vgBIO0K6Ujm8yqCkLL7ceQmZ0wiYg94wjyEwEIrFNjd9V0RC
-hE9/kJYeUC/WQegkMOwzpIPyiQO27CLodjD0CZWSqBGhHHWG23CizrjXLF+E6laLpl5jQfi5
-NCO/CmjaPvqurdhwWe0yEXNEjYJfpwC3/AEpdwTOTd08GVwXIKwzG4kCHAQQAQgABgUCVIX5
-ygAKCRAH9F4qGTfdMuAKEACeZUC5clWDnJY/XjpQXy+TI+wQyfc9hfg56agunL9+XK41W+dr
-6CGaSzY4ljPxxrtVF1Wa1bKkzvxb0WIKBGEnnbUEkdbZCaddtrE2Ctayw0W8flObBNuu8sxf
-sbzvVNvPZiMfHN0c8fgMU4+ilrWqtAWROL+qIUf3nV3CPT2X10EG1Y8WQEAF1NGtELqA3GfZ
-8Y1cqHJlqLTdiTXEUQdtfwBkWAr3FuGVIoJVEmGkvjARzCp+bwRBbBJN3fHmoOAECn1OYJkG
-cuRCvgsOeHKQW43qvNfM1qaAzARe/HLsapaL5LBh5pOSBJyEkmJRm2PomN+VdUD2hCUHp5Zv
-5Nt6vHpqRXiUg6gpGv/gczPeKVfy+kVGXJe9+c8RHogJjqizkkqE5DPfo1eq/g3QLjccAOWQ
-LW4sDlXhJ8JSop3AMyHGFbSBkHvrRLb7jFgm1A5PKeP/5xgli1NzJX9zogg6pnVG9CJoW1Rn
-q+hlPUFSRL5tNE5oCdZ4ZODPhsEWUCkWh7vuQE0zff62TrXVWfmIPgcpUvSqugNNR7VL7ULL
-7P9VnMnQfzxUz+uEZSBJZB/V5oEmJZ6dmt2iZMO4pBqzBcNhv2kNyyvM7Ca/S3FcWdUni0Z7
-COMuUZKbsTa7I+FEnNSFLQAFwUJIm6O1u6kkt5X6uE12wBoNUyxCk81XRIkCHAQQAQgABgUC
-VIX56gAKCRAWItrKlm78mAvlD/9cq0HWS9Gm5eYuI8oSJkR92FLnCYBg1rNaqjkljrktVLnG
-YSrER0zaYFHgbJI/QT1MwzgpJ6ryYMaJIhBULwZn4sY8y8X+/9ud1+9+HW5Rac6qYtOWmJ3f
-yBR3dbMJef8HRqwD8Gr2zvr7chiI3O+pJWFvdJ3sJICpA8S071rHqmVwes29FEJXlwpsiI6u
-0uL6G92ip/MAV3e3Rzd1s3zEei2uDdAGo0a2a0uwS9nh59tdRlEpGBRveTjMgfBngDG0/1H7
-crIgWdpGnoIm5mxY0rJCiJ5w38YevZGUhQYLelFWD2JKQdFJ97mJBSqmum94e4SlAsxT6GhF
-0ZDs79rqkVjROOuRcZDgB1+9uFS3cwBQi2KwWKJ/THTxGO6RRtk2+4EJPdxTuOzevlZkLUuI
-f8qSMk4tRQTX9hl5OIu7QYnZM92SwRtREUQT2hzp6/0N2AFS4U45S1Ycf2+Oa2+ZaCRqWef6
-QmgrtD7qVBw5tosA6+5c1uLtgAOkJznt0e2rRszyRV39wCrLARYqIYt5oO6eUXpmBLfihw6x
-rxM1v8CneYDyKamq5Z3sO/EC7QmyUjgv3u0vMi+aC+zxYA2qA8ndKy9kdpbClgvMHEdYYPRx
-qpFJtUpiD+gxu1zfmd82L1mmU/gUk2m2E+FvM0QSnn3yoF9mCODtGc4uh2vKK7QqV2VuZyBY
-dWV0aWFuIDxjc3NsYXllckBjaGFrcmEtcHJvamVjdC5vcmc+iQEcBBABCAAGBQJWOdY8AAoJ
-EJiN7jIV+th0bn8H/37XcjHsNR2UrqU17sGaneoHXhzPBwt7eXCm/9ZUd9hT8iqWpflZmKmb
-VHKhpDuYrIUpI85tEuxHU5Gvm5yWNpgRNTdfwVzcoS3Tt1AfTOXrbeMe2WZ/UxU6RgvtoOcx
-mSY9/7Io4WAKDpikiQIQYnJOSrHO3IOFmr6Mg9WdeJh9wmUQ0JiGKMDDtMhssPxkLOheZCa0
-KQ6kS85HUrUU3kSj7dpWH6ejc3aJ6YLlxPl4sBPIfvhT6bTwiqfQXlUsA1ALdK9Y7nON0fiR
-fE5XOePo50oIEhdsxOkkRcI6AbAwaCx2apnco6S6d4fN3P7jYc3Y20CDJRj+v+Ca56mRpYGJ
-ATgEEwECACIFAk9k+x8CGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAAAoJEI6LiYy/JBL5
-sVYIAIW8KSqlWCDrpcLVXFPRW4twvdbQ5TEQ6BjoVs8sIYSCy6o9Mp0HSPa4dLIBILJnf3Go
-GOTc0TnbsZYHmJfAjLBuR40wHA6piQYCMOh3ecKSKEfyzV8NJOZ50STJ+1F7RlE6lFP3Rnve
-ErKnofeHGKXifzNshPr7CqjktsNHySfqUoVWNu7j4huy0sT4KDQSnXtz4V+rlSFFPY2+Dv7G
-H0Qh76mOj4TspjJfp7Py/B0NBwwoHg2XPb1xiVYMatc7eL533+zb559Dczx43siSTizuKt0A
-il+61gA7/SWEOn33GbrMPSFMIAzgOhsOD2BXI+e2sZPvPfx18jLLD54U92KJAhwEEAECAAYF
-AlGXg3AACgkQAxckZKSWzeBjEBAAsXMzsNxIxjB+95K1v3QhpjPY55eTwGQWYvUXtD2rjEw3
-Y8kDuGCCT8BvqsBTjk+F18W+IofDS0054yyRC3TTXc5RTuUh3iA/KhKe6eAwA7BnmS6EFR4W
-I6rzQmMyZmMfPYtpxvW7qBfqdUO75zfYZ+/TmaNpPtGzI2kUqH86ihL6MQtjKoENeJaVacrV
-lC3I8z/++ElsPQ5+mxgiVl2zL4ZzXn5httcD5gOoodKJQ1pxXxa55RF8tl9UsX3Ee/WTY6F1
-2OxHWcECJto2axXJ10Masas2W23kGgkdynHQcN0kjHvziHXmUuv8hD1/eiFG5ClVqGDs3tBD
-upExmrzuW1MMlOZOkdkLznRehU02MAC8IVDYIBnYvg55OuK85Res/ynn0gOzXAawveW57mcY
-fFBeBV7HnxnDVIwCJD2TrUSXHrmntaQELtjz/nIcbUZgTc8HtRZIJjIsB/zLNR7ahrVlb4l1
-dvNP3aeN56PofnPsStFRTLsK06ykGF1p2vTkJ+8I3UEDTlPemCFSfUumDnyCodl8JXHHhfxj
-6dISuYaupima0lw7tYzGpzWHK5cfcaO2g/rqyS1Gwp1mKWwnCyNeeAPCWs01Jr7bL7NnRvf0
-nKvwvIiwxnR0mBHT7GKzzjQ+XxYg/t+W8nIsu2GzEy7uuje+TMyo91v3FmBH38aJAhwEEAEI
-AAYFAlSF+dYACgkQB/ReKhk33TJafw/8CL3FwBQOL2crOdUmTk6s7okNcjvAYVRRGgP7YjJ7
-qMH1WD5rAlkRJsOXJc3TyErfrq2oUYYtPKzir82VQvA1TDok598JHpqJNWMP9O/D7cXueKuU
-ahMm2sPGivtFlBSQkb2RquKmMz6byiYJTO7gu4iSziz45lb4fZ3oep18VL6SNkT4Muy6RBUW
-n/hknsJTCXp8njqWN1+J+0Ltyc1dQP44UBnrpvAlkFEsPvJuzn9Mdeh2DbB2qNxsv69/+Oob
-O75Hyv/AziEC15BTwfbRouSzB9mxfcsWnKCoqh5IzGs0M1hApVyYXVTxQ3eWKR8GKv0ahPZS
-8ImhN53uBBDzmlSIi4tWRQCIbsjwdPafHuQS0xpHgMyguGbIjUUxmbkbDkBnARkr/44CCac3
-c+YmWodyeZWZd6FjWCKPl9PNYix76wYVAcqNs/YofluquzwBJX4Af4sIW9p8b4zHFvdqF5QM
-EGfJybzBeCPHcF4ctmfCGgzpvHVv8MS0B/C6G0r2euPjwtykIB0H8iSvHLdTISJQoKxpV4Kv
-LWbaehhrJfWQclO+wDoXv9gsf3JwCvyPyXeMtjIXpcEloLMgsXq0urg12K/W8o5s3vr1ch3T
-l+Ej0dzVjd0jZbDR6wCWW3sXcyCJsAlbyQFkflQFEm9QnsRHa2fitDO6+S0BKRuKDkqJAhwE
-EAEIAAYFAlSF+fQACgkQFiLaypZu/JgDfw/8CAhxji65cFcqGrLMKjMg192f51rA3HcnFJTO
-5EZoHbJXbsESzPwZK8X6jQdah5+4jhRtZgUpLhWBOoJdttkDT7EVmIIHW8snu/Cht/CjBFMI
-5Zg6WAhsx6rVpQ+eytZ7Ix3X4c69qJxWQWFWh7IymLZ0DTy7h5xH7X1qVFl+DSgvxVoBTbXN
-N9ewjJ8ZGfeMHDsstBlC+lhpSCYQdHGa4+S5g2Gt40tv6zw97nZTBAqQDJnl5rPWUNrGNlUd
-Ehd7sJfbgHM5JJAU1fR/uqbPHxAesA8ArBug1DhO9igB5kvz4LT3P/3bhFTy3siltN2sdeEK
-C9JTQvDR231IAuFPkZv0IdvPT5gK/PeugdxhmEweDJwbrvKQ1m+upNOehctaFSHf9Fh99GV2
-vbMDR9SIOTfFNkNfrhQ96r6TVTSHiX/Rz7ZiLJl2deBYAcRptyU5DQWsucykW4DpST3x07+4
-P8VY3GRRIUNy2amS5pd89iWMZYU4R2MAj2SEQBuUrq4qo21UEzw0/A9XZFvP2/5JmseVqmCJ
-eo4D/Z1lYxRsnoim+6XTDF8VGjHdH5NdN1zb0nskBIo9ykyw/WQmlXl7AnQQnmALbwCM1IHB
-i5/DLDBTeAmslikuhCWXQRB08c/9F9RiwMC3Y5kGoTC/wvSIA+f3/VlYftTuW5rFt6fYARO5
-AQ0ETJS/CQEIAL2c7qw8gP7C7qeyHmN1mvl8qpt6BhPSA8SbYA7fOSFafk+bad4xpNlVFrV4
-3Ic3Pl406bH/ua/0gdLcwcWg3b4o5n3i7SgN9qZlhKpZzNzYJQTk2T/ew+Wgp5pojaUvXG0V
-WfYBQEJV4HrwdyGgw3rAtSNozuvjo2mcI/+H4T2mQntV7CDNBwaLT6SOSZb9T4+CQ/IYJRme
-hXfcLNVkO7vud8TrCegK4KT1cr9SxF3IxJ9BtcQLXs19jHWeWmducMbqrpXk/Av2l/B+8Qk/
-9AVAvmQ1GXIc1P5Lye/bV21icPelxC/TH8z4xPr3o5OiVTlOe9PtYDSrlR5ZTSSRqvUAEQEA
-AYkBHwQYAQIACQUCTJS/CQIbDAAKCRCOi4mMvyQS+Z1xCACJ4KTMXGz/2wRp2YM+YWy0rjsO
-+sO76PcQ4u2hxq++E9Cp/7XnIq8d9kbVug0L9Xm20jif/D5Q5ZmgggbFSWxTfSOsym30hvEh
-X64BxtT/8zC1A+lixz/J8ilOi3nGKGex0GWm9zEg+CK9AONOTr6paori8gcTTFQg3OMvwRI3
-+dkUCoOVEZRyt20lOQYY7/pTk4zPEsOZWhJdTGeFp0NPSksvJclUbZig7BUvxaM4G8v95ctc
-BAuJh5spg6X3t8gfvEmxT6JXM4z6a+kQymD8unzWFYcHXdwk9YdULriHaR3Rxahdej0ph3cl
-T4QJqjlzditPN/CwH8xZ1ZoPBpZ4
-=Olqg
------END PGP PUBLIC KEY BLOCK-----
+xsBNBEyUvwkBCAD0P0fGutq6L7iixhhOMCaxvTwmT84oEcNP816hNo2+YSEws8Xp
+6uv94czWDyC1ibzh++mStReE7RioieiKtH/sYrfS1M2Gwv3kGYzekx5qKx92nczt
+Ed08ufho3s3Qcw4QV0IT8JLdnqCWOTHwcPeyLJ0ZZH9fseqHfrlsbVIlFI7BnMZy
+TW4xTvgX1jJ9A5m5JWW70iaOX+Yom1k5WRUDm8RtzzK0IAkdXwYHbrVCRNumbQK5
+ZtNZr3EmBxDt3lzpzD1Ow1EStRpVgd0YVzHtv5N5o46Tp0L9j1br64zUbiOPrRss
+pVhpo0w9Mx70ay8JCBZqSBGM8GOSpsyP6g0FABEBAAHNKldlbmcgWHVldGlhbiA8
+Y3NzbGF5ZXJAY2hha3JhLXByb2plY3Qub3JnPsLAdgQwAQgAIBYhBCzIoGCa0qR5
+xlttXI6LiYy/JBL5BQJjrjYzAh0gAAoJEI6LiYy/JBL5hPMH/1shgXTApqi3FYQB
+sc4xZro8OolKUEYM8W6+dj3aNcEzpQGALYRCbIOt2GtJXE29VfQX8eSjQJIUI5iP
+zL/OE6IMryu9+5OWy8+oCtWht/KxHPwgoAO+UG5785yg+RFbShGjPYorHttfh5e7
+NLOOjv9sv/Y4ia1utk1YhdWsJsiwOZbZkyAmUkTh/z9sUH8bT1M1Ak7xAy7gDbLd
+HWfXXYM4wsXL6wh0mL3bPs+iAPHRv9Xt+dYL3c+bOlJqQBYET8JDHmPRp9lnbBxc
+QciYOFVJ/GRQJhKQ/FXjRtNuhjmGtmZ5u29UfQ+/2sDu6bT7K2zRyPVQyJZTj932
+FqbbzzHCwH8EMAEIACkWIQQsyKBgmtKkecZbbVyOi4mMvyQS+QUCYl5ODwsdIE9s
+ZCBlbWFpbAAKCRCOi4mMvyQS+QJCCACYq/Tch2bFTI9sFz8+IuH13KFp2RBCIb8W
+1JKaVjt/60p7ZJyy1FFBmyaJRdBxrLwAvkMhCaog/p9f9hIIHpWO0aHimZdG1xwk
+0omO3soRcLML1c0nVYpmr1Uzc6Vr13cqH4FGE8nXP0JKWlUDEazcEOxPPV6kbh2e
+WaZsoqhg4xf/8fTpGEFd8AleS8LLHpMFu9M8NmHCiaX8LQvEql0py/VA/oFvEZnF
+XSFOA6GpXQI68/tCreTwfr0dnGDvFdv/ugcW5nyvrvnEgX0PkQG5cFgj7a5EQy+E
+fPUGRU3ZLRUuHjadhZLuc0xk4ZtPIQB0+6eQkDcmYL9kMWklkBDgwsCOBBMBCgA4
+FiEELMigYJrSpHnGW21cjouJjL8kEvkFAmBeLFICGwMFCwkIBwIGFQoJCAsCBBYC
+AwECHgECF4AACgkQjouJjL8kEvnGgQgAyxPVTIq/9aegrEhgjysLEIEKnAY3+9u3
+1p99gVAqMDS71ClyYf3QBy8yAOP6Kim0YCRYG5Twazust+jn+WX9rXG+sTZBW8dz
+WAr11w9d9CqqSzctpPTksl5xY/StfmshXlj/PezP4k8sLNZyQGrsmLiKlgIOQY66
+QrbvJy2SE5BJRWHz4ZCGl5cLylV+B8anEVKDVdv+TsGQLhdUvdbZf9BHXKF5+6EF
+lkMGl76HmWJOUeuYgXbFlT3JD7Ri6Gw3v8afoHLjwreCLSfGhiv5kaZyTLd+R1pN
+NWfLp9FrxTIXmOWtRxaBp1ZOM5vMtDMzVXIlvQM7MRBH31ndoZDGycLAeAQTAQIA
+IgUCT2T7HwIbAwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AACgkQjouJjL8kEvmx
+VggAhbwpKqVYIOulwtVcU9Fbi3C91tDlMRDoGOhWzywhhILLqj0ynQdI9rh0sgEg
+smd/cagY5NzROduxlgeYl8CMsG5HjTAcDqmJBgIw6Hd5wpIoR/LNXw0k5nnRJMn7
+UXtGUTqUU/dGe94Ssqeh94cYpeJ/M2yE+vsKqOS2w0fJJ+pShVY27uPiG7LSxPgo
+NBKde3PhX6uVIUU9jb4O/sYfRCHvqY6PhOymMl+ns/L8HQ0HDCgeDZc9vXGJVgxq
+1zt4vnff7Nvnn0NzPHjeyJJOLO4q3QCKX7rWADv9JYQ6ffcZusw9IUwgDOA6Gw4P
+YFcj57axk+89/HXyMssPnhT3YsLBcwQQAQgAHRYhBKk77HoPaCz7QFww6E/JFPBl
+8t8SBQJgc+gPAAoJEE/JFPBl8t8S338P/Apk+3l3BACz9Ueu3yFtaz5l0m5a84JA
+fIX1DFx9n+N+zmDN9nyA7nfWdnCBlSY9pIhfkCIwHvqHlHYwmBJCqkg3qI3cF/za
+nUoESCL6H8a4lcKHO/raTleBSrDVD2XdtdfAYsqTdYekgfsFLFWk0P7vmr5PmK+N
+p5aggpdUwVbVvCWkW6aP5r/ODaIdkBuQ33aeWJ4mkxAFJERxdtafduupTnBUBTGP
+lJl7TSvh0dc76thfzz5dyJcN4SFAzKW+l0IeLmYv/xFr5EluVyy95r7bXg+6qBJc
+lsSSYrHKMxCVGFu5FwVR3w4nzC152oBamhp6iuD2Yz4ZyPZdEANxZVFdkvocEH/I
+KdKoBZO1XJ9S2cGVHZfpY8jip9Se0fGpgYAutq427vBvuAu11B0fHQTELFtBbkO0
+0MZ3H4U7d3VfrT12qGjxSh0RNEnTR0ztNIFBvySGgXBltsLZQlqeGAFcsQSHqaJv
+pHomXglb2v/3YOQ/tr5/V+cElE7bni72kR7pKh/ESitGqqb3AW9GJc0H6swZ7nRp
++CLhwqJF77jx/NB+TZNxwsUSEop1uVjX0AI9eRRmZeBwXnre247sZzWIpAhHEwRO
+lE/D4SHGiQ3ehEm/kDUdOoRnqmy28rOcT4H08drxGTNasC6pbK5XCZoKgEX/Z+LB
+/g+UvTc7qZqswsBcBBABCAAGBQJWOdY8AAoJEJiN7jIV+th0bn8H/37XcjHsNR2U
+rqU17sGaneoHXhzPBwt7eXCm/9ZUd9hT8iqWpflZmKmbVHKhpDuYrIUpI85tEuxH
+U5Gvm5yWNpgRNTdfwVzcoS3Tt1AfTOXrbeMe2WZ/UxU6RgvtoOcxmSY9/7Io4WAK
+DpikiQIQYnJOSrHO3IOFmr6Mg9WdeJh9wmUQ0JiGKMDDtMhssPxkLOheZCa0KQ6k
+S85HUrUU3kSj7dpWH6ejc3aJ6YLlxPl4sBPIfvhT6bTwiqfQXlUsA1ALdK9Y7nON
+0fiRfE5XOePo50oIEhdsxOkkRcI6AbAwaCx2apnco6S6d4fN3P7jYc3Y20CDJRj+
+v+Ca56mRpYHCwVwEEAEIAAYFAlSF+fQACgkQFiLaypZu/JgDfw/8CAhxji65cFcq
+GrLMKjMg192f51rA3HcnFJTO5EZoHbJXbsESzPwZK8X6jQdah5+4jhRtZgUpLhWB
+OoJdttkDT7EVmIIHW8snu/Cht/CjBFMI5Zg6WAhsx6rVpQ+eytZ7Ix3X4c69qJxW
+QWFWh7IymLZ0DTy7h5xH7X1qVFl+DSgvxVoBTbXNN9ewjJ8ZGfeMHDsstBlC+lhp
+SCYQdHGa4+S5g2Gt40tv6zw97nZTBAqQDJnl5rPWUNrGNlUdEhd7sJfbgHM5JJAU
+1fR/uqbPHxAesA8ArBug1DhO9igB5kvz4LT3P/3bhFTy3siltN2sdeEKC9JTQvDR
+231IAuFPkZv0IdvPT5gK/PeugdxhmEweDJwbrvKQ1m+upNOehctaFSHf9Fh99GV2
+vbMDR9SIOTfFNkNfrhQ96r6TVTSHiX/Rz7ZiLJl2deBYAcRptyU5DQWsucykW4Dp
+ST3x07+4P8VY3GRRIUNy2amS5pd89iWMZYU4R2MAj2SEQBuUrq4qo21UEzw0/A9X
+ZFvP2/5JmseVqmCJeo4D/Z1lYxRsnoim+6XTDF8VGjHdH5NdN1zb0nskBIo9ykyw
+/WQmlXl7AnQQnmALbwCM1IHBi5/DLDBTeAmslikuhCWXQRB08c/9F9RiwMC3Y5kG
+oTC/wvSIA+f3/VlYftTuW5rFt6fYARPCwVwEEAEIAAYFAlSF+dYACgkQB/ReKhk3
+3TJafw/8CL3FwBQOL2crOdUmTk6s7okNcjvAYVRRGgP7YjJ7qMH1WD5rAlkRJsOX
+Jc3TyErfrq2oUYYtPKzir82VQvA1TDok598JHpqJNWMP9O/D7cXueKuUahMm2sPG
+ivtFlBSQkb2RquKmMz6byiYJTO7gu4iSziz45lb4fZ3oep18VL6SNkT4Muy6RBUW
+n/hknsJTCXp8njqWN1+J+0Ltyc1dQP44UBnrpvAlkFEsPvJuzn9Mdeh2DbB2qNxs
+v69/+OobO75Hyv/AziEC15BTwfbRouSzB9mxfcsWnKCoqh5IzGs0M1hApVyYXVTx
+Q3eWKR8GKv0ahPZS8ImhN53uBBDzmlSIi4tWRQCIbsjwdPafHuQS0xpHgMyguGbI
+jUUxmbkbDkBnARkr/44CCac3c+YmWodyeZWZd6FjWCKPl9PNYix76wYVAcqNs/Yo
+fluquzwBJX4Af4sIW9p8b4zHFvdqF5QMEGfJybzBeCPHcF4ctmfCGgzpvHVv8MS0
+B/C6G0r2euPjwtykIB0H8iSvHLdTISJQoKxpV4KvLWbaehhrJfWQclO+wDoXv9gs
+f3JwCvyPyXeMtjIXpcEloLMgsXq0urg12K/W8o5s3vr1ch3Tl+Ej0dzVjd0jZbDR
+6wCWW3sXcyCJsAlbyQFkflQFEm9QnsRHa2fitDO6+S0BKRuKDkrCwVwEEAECAAYF
+AlGXg3AACgkQAxckZKSWzeBjEBAAsXMzsNxIxjB+95K1v3QhpjPY55eTwGQWYvUX
+tD2rjEw3Y8kDuGCCT8BvqsBTjk+F18W+IofDS0054yyRC3TTXc5RTuUh3iA/KhKe
+6eAwA7BnmS6EFR4WI6rzQmMyZmMfPYtpxvW7qBfqdUO75zfYZ+/TmaNpPtGzI2kU
+qH86ihL6MQtjKoENeJaVacrVlC3I8z/++ElsPQ5+mxgiVl2zL4ZzXn5httcD5gOo
+odKJQ1pxXxa55RF8tl9UsX3Ee/WTY6F12OxHWcECJto2axXJ10Masas2W23kGgkd
+ynHQcN0kjHvziHXmUuv8hD1/eiFG5ClVqGDs3tBDupExmrzuW1MMlOZOkdkLznRe
+hU02MAC8IVDYIBnYvg55OuK85Res/ynn0gOzXAawveW57mcYfFBeBV7HnxnDVIwC
+JD2TrUSXHrmntaQELtjz/nIcbUZgTc8HtRZIJjIsB/zLNR7ahrVlb4l1dvNP3aeN
+56PofnPsStFRTLsK06ykGF1p2vTkJ+8I3UEDTlPemCFSfUumDnyCodl8JXHHhfxj
+6dISuYaupima0lw7tYzGpzWHK5cfcaO2g/rqyS1Gwp1mKWwnCyNeeAPCWs01Jr7b
+L7NnRvf0nKvwvIiwxnR0mBHT7GKzzjQ+XxYg/t+W8nIsu2GzEy7uuje+TMyo91v3
+FmBH38bNH1dlbmcgWHVldGlhbiA8d2VuZ3h0QGdtYWlsLmNvbT7CwI4EEwEKADgW
+IQQsyKBgmtKkecZbbVyOi4mMvyQS+QUCYF4sUgIbAwULCQgHAgYVCgkICwIEFgID
+AQIeAQIXgAAKCRCOi4mMvyQS+RG6CADYy6b6xRwyY3e7vi4UJYQpGz0l3kfEbZVa
+wOs6qNxntU2qZ1K1wJ9xPmF/lRKO1CNZDtBOHuaJwMFKu16p/k1GPuqsTYFWrH6P
+JeJtngvIw+gPXmcJD3KXj4CamEUtbdqECkro7f31w6C5O/PkJ9E1lx2QTTHFdBTB
+XpZ7gRRVm0bWnH5KC9/sISpZnjfwsc7xJ7kDeAJW6SQIHVX/zlXDPhPbYt+ZhUWa
+lEFys6vTXDQwOCP6fhp+gIjSe2qw6zfZyA4WTRzr2DHUXmAx1ogH//7qITcdG/mo
+CsHR2X6/nZ/WH0/EyQyLoR091Xt6gJxx8QcRW7V0GFvxSSRBT0nTwsB7BBMBAgAl
+AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAUCT2T8OQIZAQAKCRCOi4mMvyQS
++RAyB/9h9J2amfLGMDtVqBepH78SQQ6LbLmMQzgF6kPDM0telxR2ryMhY3mJxUBw
+20xxszCzwE0phG/B4whYBH7xyCWI1+e7n2SO5rfSoO/Y40RSKOuScL7RWFcVB87R
+0Ao2AizlPBKp5uhSpIKY0NcS+UC5z3wW3rVGL+S5NgulybYThxDv6AAr7dPdqVNk
+ILuXjLZDAYoWgs8L6fXpiB48tGiTvj8NQdsdqaX3ZkWDz8sXdhY4hdSRHvROVCBL
+CjKReKi4kqjULpRDqvQ4f/l6ux6fg8vc/C8TGWDMDgCJsBC/QTtD7yef6YlISvIr
+cmnwWtEUqIyBx2Kn+ILddm9mkNmywsB4BBMBAgAiBQJMlL8JAhsDBgsJCAcDAgYV
+CAIJCgsEFgIDAQIeAQIXgAAKCRCOi4mMvyQS+drgCADA87rxdB2nLgjy8Xa3T9Bw
+9Sh/ELDsx7ijAF+coNXhBpnqgghv5IlUFXd/kea5pV4KVer5fRaAg5VrmCWJWzCw
+SAsA18/6dRiarhbWgKq2GgEoP9C/t2VMtJtb7Sjg2MQbnCoj8kluhrBKa8TVly1l
+jDHIlvVJhgJAHxEG0xmqsbBeDeOvcdnBCeomlK60U9N0J42pqNqdXMtoPN5upYj7
+dQoNxqWI9QuFYUKRHHS0mLmb8s97VfrhWYtZyCP1pDoUOWuc2ufyMaj+F6BcB9aZ
+vgr3OCkWnDW0BQL0JFWzpQ/gLm9jftms2mAWMZL1sTY+DaymCVqKiWWJBeMmVxX1
+wsFzBBABCAAdFiEEqTvseg9oLPtAXDDoT8kU8GXy3xIFAmBz6A8ACgkQT8kU8GXy
+3xIRxBAA4GVYvYcQAPeBTQSqo7wAEzgxT2Be8U3tLLmCFwOy0zqudTi9DfXRlX2/
+UDGc8OZzN0J+EBVq+V/m3+BYhqkSsyFwMdUL+gJys2a1/thtWtl5Ntd7GDW7z9ww
+hJyrZHruLsvaOX2hRc/MpFkSFG8xDdE+uXIj3VBvf0lp38c/sE+SOuu/FpDcxqao
+ZNk7W4fLyKGV28twr1sw2R8ngcLl71hLUnJ5xPXti2JY89xua/A6Jpy22Zfn9wgu
+e3T9sQmkJNg3v22xwiEVXzd5qk9f7Ox7pLCiq3xHHq1Cir3U1v3hmYJHQv+cCeyc
+LIv2NoiNYsyBNYn8YvWoLecGx5P2t02rMEzE3tF9MEjndYIoTp8kiajaQiLXplAv
+YsA1LZ2oYMm09+dxJcWa4Ar1kbn+wR260IdM1WHa1DxVvfkPvW3dAhU8eFDK3npk
+2zs8qB8kWnXjBgWRL+SbWeTuanQYSx5siqci7efnR4gWn8AGGbfThA8Hq9ZHpIDT
+gcUre9mG68hahCs/VdtcUV94OS8Kw6HxS8yhEDz500Cy/6KsANOdKuAyH7lRky6N
+zE10y6EtGhk/yk77cOJH1D7z5ogsjkedvXJCFMFFX5CGT8wJewmCVI4bcuOAXr9N
+v4YvtzHbe2jdYoyLGe0PpxCepEcVYo99LMXWNUPJs9plRIoKeXTCwFwEEAEIAAYF
+AlY51jwACgkQmI3uMhX62HQvewf+PYNTp9CEEC38kB4FmHhMdwF4FCqnkAJN9V1I
+Nw2q07Z1fpZSiDddTcPf6MvOtZ3TmTAjtBes5q5UilJwfN8l+7BJA2KBV5vVVuIA
+Ehu0qzcoxPJNhu7XJD7xZH1xoGc40pnUwfc2pcTj92+HSrdI8NAyUC0mtBjRs8IV
+5yLXVfmGzxZECv7R6f50Wh2eM8xPEzrkIPJbpSb/OmYvmUAUJ5o0xU9YPPmAA4jm
+HeIkSoXJvV30uOULt/a0Ygk0Qx6COpnLRFrc6JZ+0LY1Sye82tqC6RQfegvvlCRN
+ZEOGDu6eCX9ZYeHaXA6kin6xXDXKp+2jPL7Wf6I7iqVLKBJfdsLBXAQQAQgABgUC
+VIX56gAKCRAWItrKlm78mAvlD/9cq0HWS9Gm5eYuI8oSJkR92FLnCYBg1rNaqjkl
+jrktVLnGYSrER0zaYFHgbJI/QT1MwzgpJ6ryYMaJIhBULwZn4sY8y8X+/9ud1+9+
+HW5Rac6qYtOWmJ3fyBR3dbMJef8HRqwD8Gr2zvr7chiI3O+pJWFvdJ3sJICpA8S0
+71rHqmVwes29FEJXlwpsiI6u0uL6G92ip/MAV3e3Rzd1s3zEei2uDdAGo0a2a0uw
+S9nh59tdRlEpGBRveTjMgfBngDG0/1H7crIgWdpGnoIm5mxY0rJCiJ5w38YevZGU
+hQYLelFWD2JKQdFJ97mJBSqmum94e4SlAsxT6GhF0ZDs79rqkVjROOuRcZDgB1+9
+uFS3cwBQi2KwWKJ/THTxGO6RRtk2+4EJPdxTuOzevlZkLUuIf8qSMk4tRQTX9hl5
+OIu7QYnZM92SwRtREUQT2hzp6/0N2AFS4U45S1Ycf2+Oa2+ZaCRqWef6QmgrtD7q
+VBw5tosA6+5c1uLtgAOkJznt0e2rRszyRV39wCrLARYqIYt5oO6eUXpmBLfihw6x
+rxM1v8CneYDyKamq5Z3sO/EC7QmyUjgv3u0vMi+aC+zxYA2qA8ndKy9kdpbClgvM
+HEdYYPRxqpFJtUpiD+gxu1zfmd82L1mmU/gUk2m2E+FvM0QSnn3yoF9mCODtGc4u
+h2vKK8LBXAQQAQgABgUCVIX5ygAKCRAH9F4qGTfdMuAKEACeZUC5clWDnJY/XjpQ
+Xy+TI+wQyfc9hfg56agunL9+XK41W+dr6CGaSzY4ljPxxrtVF1Wa1bKkzvxb0WIK
+BGEnnbUEkdbZCaddtrE2Ctayw0W8flObBNuu8sxfsbzvVNvPZiMfHN0c8fgMU4+i
+lrWqtAWROL+qIUf3nV3CPT2X10EG1Y8WQEAF1NGtELqA3GfZ8Y1cqHJlqLTdiTXE
+UQdtfwBkWAr3FuGVIoJVEmGkvjARzCp+bwRBbBJN3fHmoOAECn1OYJkGcuRCvgsO
+eHKQW43qvNfM1qaAzARe/HLsapaL5LBh5pOSBJyEkmJRm2PomN+VdUD2hCUHp5Zv
+5Nt6vHpqRXiUg6gpGv/gczPeKVfy+kVGXJe9+c8RHogJjqizkkqE5DPfo1eq/g3Q
+LjccAOWQLW4sDlXhJ8JSop3AMyHGFbSBkHvrRLb7jFgm1A5PKeP/5xgli1NzJX9z
+ogg6pnVG9CJoW1Rnq+hlPUFSRL5tNE5oCdZ4ZODPhsEWUCkWh7vuQE0zff62TrXV
+WfmIPgcpUvSqugNNR7VL7ULL7P9VnMnQfzxUz+uEZSBJZB/V5oEmJZ6dmt2iZMO4
+pBqzBcNhv2kNyyvM7Ca/S3FcWdUni0Z7COMuUZKbsTa7I+FEnNSFLQAFwUJIm6O1
+u6kkt5X6uE12wBoNUyxCk81XRMLBXAQQAQIABgUCUZeDcAAKCRADFyRkpJbN4HNJ
+EACH+Vbdn5iJcwI41xc2x6ZiRR/FXXymNYuGtDd2NJlFuaL22fyxyacEWdD7t70l
+6diGCWVvR/696TjeJQojPOkTERAT++34RxdlM5ZTSah0JUWfzgvoTL9KoD0InYA+
+ALraX3zZHOKTf5jnhyqwujGXi3wunntF4jcxw5fMk+U3mfpMwniVs9o0BOVmfRWM
+jia4TTPrWUbizVY/hhbOXgUsHdYVj0GbLAvSj8IUVY/XerM+Zx/bZ54Hto9A5h8e
+o6jo5jIJOrAClh+Vgh4Ind7V9jgq1Upxi3fKp2t/D2ZZu/hib5qJKorAW3Sw2gZx
+K4gNv1AdBNPKfgYk7HHZP/kJKNwtl3pv/EE/R0oFxVaUHCOxe4GzIHHNXWxDCKUS
+5kElLhGnxFG//ZmpZ2Kya6+4FcsZVxNStEoPR/DUtdtRMDO36DTMoDVHqIXaMJT2
+50H24sKwB0zKD2vVKcvAisQdwC+hmLCAFV8c30aA2ijKlQ3HTqR08r54eRopO5T9
+vgBIO0K6Ujm8yqCkLL7ceQmZ0wiYg94wjyEwEIrFNjd9V0RChE9/kJYeUC/WQegk
+MOwzpIPyiQO27CLodjD0CZWSqBGhHHWG23CizrjXLF+E6laLpl5jQfi5NCO/Cmja
+PvqurdhwWe0yEXNEjYJfpwC3/AEpdwTOTd08GVwXIKwzG87ATQRMlL8JAQgAvZzu
+rDyA/sLup7IeY3Wa+Xyqm3oGE9IDxJtgDt85IVp+T5tp3jGk2VUWtXjchzc+XjTp
+sf+5r/SB0tzBxaDdvijmfeLtKA32pmWEqlnM3NglBOTZP97D5aCnmmiNpS9cbRVZ
+9gFAQlXgevB3IaDDesC1I2jO6+OjaZwj/4fhPaZCe1XsIM0HBotPpI5Jlv1Pj4JD
+8hglGZ6Fd9ws1WQ7u+53xOsJ6ArgpPVyv1LEXcjEn0G1xAtezX2MdZ5aZ25wxuqu
+leT8C/aX8H7xCT/0BUC+ZDUZchzU/kvJ79tXbWJw96XEL9MfzPjE+vejk6JVOU57
+0+1gNKuVHllNJJGq9QARAQABwsBfBBgBAgAJBQJMlL8JAhsMAAoJEI6LiYy/JBL5
+nXEIAIngpMxcbP/bBGnZgz5hbLSuOw76w7vo9xDi7aHGr74T0Kn/tecirx32RtW6
+DQv1ebbSOJ/8PlDlmaCCBsVJbFN9I6zKbfSG8SFfrgHG1P/zMLUD6WLHP8nyKU6L
+ecYoZ7HQZab3MSD4Ir0A405OvqlqiuLyBxNMVCDc4y/BEjf52RQKg5URlHK3bSU5
+Bhjv+lOTjM8Sw5laEl1MZ4WnQ09KSy8lyVRtmKDsFS/Fozgby/3ly1wEC4mHmymD
+pfe3yB+8SbFPolczjPpr6RDKYPy6fNYVhwdd3CT1h1QuuIdpHdHFqF16PSmHdyVP
+hAmqOXN2K0838LAfzFnVmg8Glng=
+=8BEE
+-----END PGP PUBLIC KEY BLOCK-----
diff --git a/fcitx5-libthai.spec b/fcitx5-libthai.spec
index 514e8da..a58c97d 100644
--- a/fcitx5-libthai.spec
+++ b/fcitx5-libthai.spec
@@ -11,7 +11,7 @@ Source:     https://download.fcitx-im.org/fcitx5/%{name}/%{name}-%{version}.tar.
 Source1:    https://download.fcitx-im.org/fcitx5/%{name}/%{name}-%{version}.tar.zst.sig
 Source2:    https://pgp.key-server.io/download/0x8E8B898CBF2412F9
 
-BuildRequires:  gnupg2
+BuildRequires:  openpgpverify
 BuildRequires:  cmake
 BuildRequires:  extra-cmake-modules
 BuildRequires:  gcc-c++
@@ -27,7 +27,7 @@ Requires:       fcitx5-data
 %{summary}.
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %autosetup
 
 %build
-- 
2.56.0

